doc/release-process.md

```diff
@@ -101,8 +103,9 @@ Run `./fetch-params.sh`.
 
 ## D. Make tag for the newly merged result
 
-In this example, we ensure master is up to date with the
-previous merged PR, then:
+Checkout master and pull the latest version to ensure master is up to date with the release PR which has recently been reviewed and merged. Check the last commit on the local and remote versions of master to make sure they are the same. 
```

Comment:
- `s/which has recently been reviewed and merged/merged in B4/`
- Break the line?